controllers/depresolver/depresolver_config.go

```diff
@@ -94,6 +94,9 @@ func (dr *DependencyResolver) validateConfig(config *Config) (err error) {
 	if config.Log.Format == NoFormat {
 		return fmt.Errorf("invalid '%s', allowed values ['','%s','%s']", LogFormatKey, JSONFormat, SimpleFormat)
 	}
+	if config.EdgeDNSType == DNSTypeError {
+		return fmt.Errorf("more than one EdgeDNS types recognised")
```

Comment:
@kuritka, It would be good to help user with information on which providers have been detected in configuration.

Reply:
@somaritane , agreed, implemented, although it required more changes, primarily: [[1]](https://github.com/AbsaOSS/k8gb/pull/450/files#diff-ef477f235dc201dc9a8f7505e61107d1be845c695ed8d201e07939e2496ec2edR66-R75), [[2]](https://github.com/AbsaOSS/k8gb/pull/450/files#diff-b94e4b6c67fca17dce52708f765679ec46a8553531b432d30b8da1b11b1bc89cR83-R84)